drydock/patches/uwsgi-config

```diff
@@ -0,0 +1,14 @@
+# Fix 502 errors for closed connections
+http-keepalive = 1
+add-header = Connection: Keep-Alive
+
+# Better startup/shutdown in docker:
+die-on-term = true
+lazy-apps = false
+need-app = true
+no-defer-accept = true
```

Comment:
Can we apply this with low priority somehow? I'm thinking that an inline plugin may want to add their own overrides for this and it should be able to.

Reply:
Inline plugins are applied after plugins:

```patch
diff --git a/environments/chennai/config.yml b/environments/chennai/config.yml
index 2a052f2..a217d7e 100644
--- a/environments/chennai/config.yml
+++ b/environments/chennai/config.yml
@@ -108,6 +108,7 @@ PLUGINS:
 - notes
 - pod-autoscaling
 - pythonsettings
+- uwsgi-tunning
 PLUGIN_INDEXES:
 - https://overhang.io/tutor/main
 RALPH_ADMIN_HASHED_PASSWORD: $2b$12$REkmjVDdk8DDf5pJmmcoh.JjWXxl8hzfhZ55DByFsAQAwzgUTGLuy
diff --git a/environments/chennai/env/apps/openedx/uwsgi.ini b/environments/chennai/env/apps/openedx/uwsgi.ini
index 1d0022b..7600b2a 100644
--- a/environments/chennai/env/apps/openedx/uwsgi.ini
+++ b/environments/chennai/env/apps/openedx/uwsgi.ini
@@ -22,3 +22,4 @@ no-defer-accept = true
 master = true
 py-call-osafterfork = true
 vacuum = true
+die-on-term = false
diff --git a/environments/chennai/plugins/uwsgi-tunning.yml b/environments/chennai/plugins/uwsgi-tunning.yml
index 00db159..6582be2 100644
--- a/environments/chennai/plugins/uwsgi-tunning.yml
+++ b/environments/chennai/plugins/uwsgi-tunning.yml
@@ -1,3 +1,5 @@
 name: uwsgi-tunning
 patches:
-  uwsgi-config: |
\ No newline at end of file
+  uwsgi-config: |
+    die-on-term = false
```

Reply:
IRRC they are applied in alphabetical order, try renaming your plugin to `01-uwsgi-tunning`

Reply:
I've change it locally to `aaaa` and still works

Reply:
This is an annoying edge case: yaml are applied later, but python modules and python packages are applied at the same time.